src/Features/Core/Portable/AddImport/AddImportFixData.cs

```diff
@@ -4,14 +4,17 @@
 
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.Runtime.Serialization;
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.Tags;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.AddImport
 {
-    internal class AddImportFixData
+    [DataContract]
+    internal sealed class AddImportFixData
     {
+        [DataMember(Order = 0)]
```

Comment:
❔ These contracts look super easy to mess up. What analyzer(s) do we currently have for these?